### PR TITLE
EXLM-152 Video block

### PR DESCRIPTION
**Pull request by ruhisingh1**
Please always provide the Jira Issue your PR is for, as well as test URLs where your change can be observed (before and after):

Jira ID: [EXLM-152](https://jira.corp.adobe.com/browse/EXLM-152)

Test URLs:

- Before: https://main--exlm--adobe-experience-league.hlx.page/docs/authoring-guide-exl/using/markdown/cheatsheet
- After: https://feat-exlm55-video--exlm--adobe-experience-league.hlx.page/docs/authoring-guide-exl/using/markdown/cheatsheet

### CODE REVIEW

**Review by tprasad05 — APPROVED** [review]

File: blocks/embed/embed.scss [hunk 1]
```diff
@@ -0,0 +1,74 @@
+main {
```
Comment (noormd08) [inline blocks/embed/embed.scss:1]: You can remove this main reference.

File: blocks/embed/embed.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+					box-sizing: border-box;
+					position: absolute;
+					width: 0;
+					height: 10px;
+					border-top: 5px solid transparent;
+					border-bottom: 5px solid transparent;
+					border-left: 6px solid;
+					top: 4px;
+					left: 7px;
+				}
```
Comment (noormd08) [inline blocks/embed/embed.scss:62]: By default, button has specific styles. We can override only those fields and move all the button css to generic styles.

File: blocks/embed/embed.scss [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+				margin-right: auto;
+			}
+		}
+		.embed-placeholder {
+			width: 100%;
+			aspect-ratio: 16 / 9;
+			position: relative;
+			& > * {
+				display: flex;
+				align-items: center;
```
Comment (thomsebastin) [inline blocks/embed/embed.scss:23]: @ruhisingh1 Looks like the indentation is off and also if possible try to reduce the level of nesting. Less nesting looks more readable imo.

File: blocks/embed/embed.scss [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+			}
+		}
+	}
+	.embed-video {
+		left: 0;
+		width: 100%;
+		height: 0;
+		position: relative;
+		padding-bottom: 56.25%;
+		border: 1px solid #1e1e1e;
```
Comment (thomsebastin) [inline blocks/embed/embed.scss:72]: Please check if we can use variables here.

File: blocks/embed/embed.scss (hunk 2 above)
Comment (ruhisingh1) [reply blocks/embed/embed.scss]: This CSS is specific to video play button

File: blocks/embed/embed.scss (hunk 3 above)
Comment (ruhisingh1) [reply blocks/embed/embed.scss]: Done

File: blocks/embed/embed.scss (hunk 4 above)
Comment (ruhisingh1) [reply blocks/embed/embed.scss]: Didn't find this color variable in styles.css

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**